src/main/java/org/galatea/starter/service/WorldTimeService.java

```diff
@@ -0,0 +1,38 @@
+package org.galatea.starter.service;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import net.sf.aspect4log.Log;
+import org.galatea.starter.domain.WorldTimeApiResponse;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Slf4j
+@Log
+@Service
+public class WorldTimeService {
+
+  @NonNull
+  private WorldTimeClient worldTimeClient;
+
+  // Produces in the format of "Wed, Dec 22 2021 04:24 PM"
+  private static final DateTimeFormatter DATE_TIME_FORMATTER =
+      DateTimeFormatter.ofPattern("EEE, MMM dd yyyy h:mm a");
+
+  /**
+   * Get the current time for the given timezone from the World Time API (see WorldTimeClient).
+   *
+   * @param timezone the timezone to return the current time for.
+   * @return A string of the current time (see DATE_TIME_FORMATTER field for format).
+   */
+  public String getCurrentTimeForTimezone(final String timezone) {
```

Comment:
Can't java do this without the remote call?

Reply:
@cbaltera Yeah that makes way more sense. Was looking at a guide on setting up wit.ai integration and took this example and didn't think much about improving it. I've updated the PR to do this now with ZonedDateTime.